backend/coreapp/serializers.py

```diff
@@ -99,6 +97,7 @@ class ScratchSerializer(serializers.ModelSerializer["Scratch"]):
     slug = serializers.SlugField(read_only=True)
     url = UrlField()
     html_url = HtmlUrlField()
+    parent = HtmlUrlField()
```

Comment:
Pretty sure this should be UrlField, not HtmlUrlField

Reply:
why? it wants to be a link the user clicks, not the API link

Reply:
No - see `AboutScratch.tsx`; the fork link is `<ScratchLink url={scratch.parent} />`, and ScratchLink makes an API request to the given `url` parameter to gather information about the scratch (such as its name and html_url).

Reply:
that's odd, cause it worked for me locally